test/unit/uk/gov/hmrc/exports/scheduler/PurgeDraftDeclarationsJobSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.exports.scheduler
+
+import java.time._
+
+import org.mockito.BDDMockito.given
+import org.mockito.Mockito.{reset, verify}
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.mockito.MockitoSugar
+import uk.gov.hmrc.exports.config.{AppConfig, JobConfig}
+import uk.gov.hmrc.exports.repositories.DeclarationRepository
+import uk.gov.hmrc.exports.scheduler.PurgeDraftDeclarationsJob
+import unit.uk.gov.hmrc.exports.base.UnitSpec
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class PurgeDraftDeclarationsJobSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach {
+
+  private val zone = ZoneOffset.UTC
+  private val clock: Clock = Clock.fixed(Instant.now(), zone)
+  private val appConfig = mock[AppConfig]
+  private val declarationRepository = mock[DeclarationRepository]
+
+  override def afterEach(): Unit = {
+    super.afterEach()
+    reset(appConfig, declarationRepository)
+    given(appConfig.clock) willReturn clock
```

Comment:
We're using `when` `thenReturn` for unit specs, but it's not an issue

Reply:
I don't think we should be too picky about this kind of thing, both are readable and theres no real benefit of being consistent